src/main/java/gravitas/command/UpdateTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+package gravitas.command;
+
+import gravitas.exception.DukeException;
+import gravitas.storage.Storage;
+import gravitas.task.Task;
+import gravitas.tasklist.TaskList;
+
+/**
+ * Command to update a task.
+ */
+public class UpdateTaskCommand extends Command {
+
+    private static final String OUT_OF_BOUND = "The task that you wish to mark is invalid! please try again!";
```

Comment:
Just a suggestion to rename OUT_OF_BOUND as INVALID_TASK_MESSAGE. MARK as TASK_MARKED_MESSAGE and UNMARK as TASK_UNMARKED_MESSAGE for betting naming convention